Snapshot/community_ab_test_pipeline.json

```diff
@@ -0,0 +1,1457 @@
+{
+  "metadata": {
```

Comment:
Like mentioned in our slack conversation, would be interesting in later tests to see how to break down query pipeline JSONs to break it up into smaller files for each section of the query pipeline (easier for PR reviews)